### PR TITLE
fixing 404 city data

### PR DESCRIPTION
**Pull request by alarangeiras**
fixing 404 city data

### CODE REVIEW

**Comment by robertoduessmann** [comment]
Thanks for you contribution @alarangeiras , looks like very nice :smiley:

File: controller/weather.go [hunk 1]
```diff
@@ -47,23 +55,44 @@ func getExternalWeather(city string) *http.Response {
...
 		weather.Forecast[i].Day = i + 1
 		weather.Forecast[i].Temperature = parser.Parse(doc, temperatureForecastTags[i]) + " °C"
 		weather.Forecast[i].Wind = parser.Parse(doc, windForecastTags[i]) + " km/h"
 	}
+
+	return nil
+}
+
+func notFound(weather *model.Weather) bool {
+	if len(weather.Description) == 0 && len(weather.Description) == 0 && len(weather.Description) == 0 {
```
Comment (robertoduessmann) [inline controller/weather.go]: I only didn't catch why you checked 3 times here, the conditions are exactly the same right? Could be just once?

Comment (alarangeiras) [reply controller/weather.go]: Yes, you're right.
In fact this was the remnant of another idea that I had to check three different fields.
But in the end I figured that just checking the Description field was enough.
I've already pushed the fix.

Thanks for the remark

**Comment by alarangeiras** [comment]
Let me know if I can help with any other assignments.
Or other initiatives you may have.

File: controller/weather.go (hunk 1 above)
Comment (robertoduessmann) [reply controller/weather.go]: Oh I see, perfect :ok_hand:

**Review by robertoduessmann — APPROVED** [review]

**Comment by robertoduessmann** [comment]
Thank u so much @alarangeiras :tada: 
There are more some interesting issues if you want take a look (forecast with date, get temperature in F or C, ...). 
And also I have this another project in Go https://github.com/robertoduessmann/health-check to check the server "health", if you're interested.
Thank you again :smile:

**Comment by alarangeiras** [comment]
Sure, I'm interested in helping out with other projects.
I'll take a look in these issues too.

Thanks

**Pull request merged**